src/code/views/local-file-tab-save-view.coffee

```diff
@@ -55,8 +56,9 @@ module.exports = React.createClass
 
   confirm: (e, simulateClick) ->
     if not @confirmDisabled()
-      @refs.download.href = @props.client.getDownloadUrl(@state.content, @state.includeShareInfo, @state.mimeType)
-      @refs.download.click() if simulateClick
+      blob = @props.client.getDownloadBlob(@state.content, @state.includeShareInfo, @state.mimeType)
+      FileSaver.saveAs(blob, @state.downloadFilename, true)
```

Comment:
Do we need to move the preventDefault from the else below to above the if since we are no longer using the href transform?